server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/endpoint/GetAssertionsTests.java

```diff
@@ -0,0 +1,248 @@
+        tinyIRVContestResult.setBallotCount(10L);
+        tinyIRVContestResult.setWinners(Set.of("Alice"));
+        tinyIRVContestResult.addContests(Set.of(tinyIRVExample));
+
+        wireMockRaireServer.start();
+        configureFor("localhost", wireMockRaireServer.port());
+        stubFor(post(urlEqualTo("/raire/get-assertions-csv"))
+                .willReturn(aResponse()
+                        .withStatus(200)
+                        .withHeader("Content-Type", "application/octet-stream")
+                        .withBody("Test csv")));
+        stubFor(post(urlEqualTo("/raire/get-assertions-json"))
+                .willReturn(aResponse()
+                .withStatus(200)
+                .withHeader("Content-Type", "application/json")
+                .withBody("Test json")));
+    }
+
+    @AfterClass
+    public void closeMocks() {
+        wireMockRaireServer.stop();
+    }
+
+    /**
+     * Calls the getAssertions main endpoint function, for csv, and checks that the right file names are present in the
+     * zip.
+     * @throws Exception never.
+     */
+    @Test
+    public void rightFileNamesInZipCSV() throws Exception {
+        testUtils.log(LOGGER, "rightFileNamesInZipCSV");
+
+        try (MockedStatic<IRVContestCollector> mockIRVContestResults = Mockito.mockStatic(IRVContestCollector.class)) {
+            mockIRVContestResults.when(IRVContestCollector::getIRVContestResults).thenReturn(mockedIRVContestResults);
+
+            GetAssertions endpoint = new GetAssertions();
+            ByteArrayOutputStream bytesOut = new ByteArrayOutputStream();
+            ZipOutputStream zos = new ZipOutputStream(bytesOut);
+            endpoint.getAssertions(zos, BigDecimal.valueOf(0.03), wireMockRaireServer.baseUrl()+"/raire/get-assertions", "csv");
+
+            byte[] bytes = bytesOut.toByteArray();
+            InputStream bais = new ByteArrayInputStream(bytes);
+            ZipInputStream in = new ZipInputStream(bais);
+            ZipEntry firstEntry = in.getNextEntry();
+            assertNotNull(firstEntry);
+            assertEquals(firstEntry.getName(), "CityofBoulderMayoralCandidates_assertions.csv");
+            ZipEntry secondEntry = in.getNextEntry();
+            assertEquals("TinyExample1" + "_assertions.csv", secondEntry.getName());
+            ZipEntry thirdEntry = in.getNextEntry();
+            assertNull(thirdEntry);
+        }
+    }
+
+    /**
+     * Calls the getAssertions main endpoint function, for json, and checks that the right file names are present in the
+     * zip.
+     * @throws Exception never.
+     */
+    @Test
+    public void rightFileNamesInZipJSON() throws Exception {
+        testUtils.log(LOGGER, "rightFileNamesInZipJSON");
+        try (MockedStatic<IRVContestCollector> mockIRVContestResults = Mockito.mockStatic(IRVContestCollector.class)) {
+            mockIRVContestResults.when(IRVContestCollector::getIRVContestResults).thenReturn(mockedIRVContestResults);
+
```

Comment:
Not sure about the hardcoded localhost url, there already is a URL set in the test properties.